R/zchunk_LB181.ag_R_C_Y_GLU_irr_mgmt.R

```diff
@@ -52,83 +54,351 @@ module_aglu_LB181.ag_R_C_Y_GLU_irr_mgmt_DISABLED <- function(command, ...) {
-    # If no precursors (very rare) don't call `add_precursor` at all
-    tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
+    L181.LC_bm2_R_C_Yh_GLU_irr_level %>%
+      add_title("Cropland cover by GCAM region / commodity / year / GLU / irrigation / mgmt level") %>%
+      add_units("bm2") %>%
+      add_comments("Cropland cover by high and low management levels are currently set at the share of 50% by each.") %>%
       add_legacy_name("L181.LC_bm2_R_C_Yh_GLU_irr_level") %>%
-      add_precursors("precursor1", "precursor2", "etc") %>%
-      # typical flags, but there are others--see `constants.R`
-      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
+      add_precursors("common/iso_GCAM_regID",
+                     "aglu/LDS/Mueller_yield_levels",
+                     "aglu/Muller_crops",
+                     "aglu/FAO_ag_items_PRODSTAT",
+                     "L151.ag_irrHA_ha_ctry_crop",
+                     "L151.ag_rfdHA_ha_ctry_crop",
+                     "L151.ag_irrProd_t_ctry_crop",
+                     "L151.ag_rfdProd_t_ctry_crop",
+                     "L171.LC_bm2_R_rfdHarvCropLand_C_Yh_GLU",
+                     "L171.LC_bm2_R_irrHarvCropLand_C_Yh_GLU") %>%
+      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR, FLAG_PROTECT_FLOAT) ->
       L181.LC_bm2_R_C_Yh_GLU_irr_level
-    tibble() %>%
-      add_title("descriptive title of data") %>%
-      add_units("units") %>%
-      add_comments("comments describing how data generated") %>%
-      add_comments("can be multiple lines") %>%
+
+    L181.ag_EcYield_kgm2_R_C_Y_GLU_irr_level %>%
+      add_title("Economic yield by GCAM region / commodity / year / GLU / irrigation / mgmt level") %>%
+      add_units("kg/bm2") %>%
+      add_comments("Economic yields are calculated as the observed yields times a mutiplier for high or low mgmt level.") %>%
+      add_comments("Currently the same yield mutipliers are set for all region/commodity/GLU/irrigation.") %>%
       add_legacy_name("L181.ag_EcYield_kgm2_R_C_Y_GLU_irr_level") %>%
-      add_precursors("precursor1", "precursor2", "etc") %>%
-      # typical flags, but there are others--see `constants.R`
-      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR) ->
+      add_precursors("common/iso_GCAM_regID",
+                     "aglu/LDS/Mueller_yield_levels",
+                     "aglu/Muller_crops",
+                     "aglu/FAO_ag_items_PRODSTAT",
+                     "L151.ag_irrHA_ha_ctry_crop",
+                     "L151.ag_rfdHA_ha_ctry_crop",
+                     "L151.ag_irrProd_t_ctry_crop",
+                     "L151.ag_rfdProd_t_ctry_crop",
+                     "L171.ag_irrEcYield_kgm2_R_C_Y_GLU",
+                     "L171.ag_rfdEcYield_kgm2_R_C_Y_GLU") %>%
+      add_flags(FLAG_LONG_YEAR_FORM, FLAG_NO_XYEAR, FLAG_SUM_TEST) ->
```

Comment:
One observation failed the test and the diff is `4.285200e-11`, and it passed the `FLAG_SUM_TEST`.